app/controllers/consultations_controller.rb

```diff
@@ -52,6 +52,12 @@ def create
     end
   end
 
+  def destroy
+    authorize @consultation
+    @consultation = Consultation.find(params[:id])
```

Comment:
In my understanding, this is not needed? Because before_action :delete, find_consultation runs, which looks for a consultation by id. It might be redundant.

Reply:
You might be right. However, my code wasn't working and I made some changes and it seemed to work. I'm not sure which changes were the successful ones! Let's check on Thursday.